### PR TITLE
Unknown segments removing.

### PR DESCRIPTION
**Pull request by darina**

### CODE REVIEW

**Comment by darina** [comment]
@rokuz @bykoianko PTAL

**Comment by rokuz** [comment]
LGTM

File: traffic/traffic_info.cpp [hunk 1]
```diff
@@ -495,7 +495,8 @@ bool TrafficInfo::UpdateTrafficData(vector<SpeedGroup> const & values)
   }
 
   for (size_t i = 0; i < m_keys.size(); ++i)
-    m_coloring.emplace(m_keys[i], values[i]);
+    if (values[i] != SpeedGroup::Unknown)
+      m_coloring.emplace(m_keys[i], values[i]);
```
Comment (mpimenov) [inline traffic/traffic_info.cpp]: {
}

File: routing/routing_session.cpp [hunk 2]
```diff
@@ -599,7 +599,8 @@ void RoutingSession::OnTrafficInfoAdded(TrafficInfo && info)
   TrafficInfo::Coloring coloring;
   for (auto const & kv : fullColoring)
   {
-    if (kv.second != SpeedGroup::G5 && kv.second != SpeedGroup::Unknown)
+    ASSERT(kv.second != SpeedGroup::Unknown, ());
```
Comment (mpimenov) [inline routing/routing_session.cpp]: ASSERT_NOT_EQUAL ?

**Comment by mpimenov** [comment]
LGTM after fixes

File: drape_frontend/drape_engine.cpp [hunk 3]
```diff
@@ -547,13 +547,13 @@ void DrapeEngine::UpdateTraffic(traffic::TrafficInfo const & info)
...
   for (auto const & segmentPair : info.GetColoring())
-  {
-    if (segmentPair.second != traffic::SpeedGroup::Unknown)
-      mwmColoring.insert(segmentPair);
-  }
+    ASSERT(segmentPair.second != traffic::SpeedGroup::Unknown, ());
+#endif
+
+  df::TrafficSegmentsColoring segmentsColoring;
+  segmentsColoring.insert(make_pair(info.GetMwmId(), info.GetColoring()));
```
Comment (bykoianko) [inline drape_frontend/drape_engine.cpp]: #include "std/utility.hpp" for make_pair pls.

File: traffic/traffic_info.cpp [hunk 4]
```diff
@@ -495,7 +495,8 @@ bool TrafficInfo::UpdateTrafficData(vector<SpeedGroup> const & values)
   }
 
   for (size_t i = 0; i < m_keys.size(); ++i)
```
Comment (bykoianko) [inline traffic/traffic_info.cpp]: Add {} pls.

File: drape_frontend/drape_engine.cpp (hunk 3 above)
Comment (bykoianko) [inline drape_frontend/drape_engine.cpp]: emplace?

**Comment by darina** [comment]
@bykoianko PTAL

**Comment by bykoianko** [comment]
LGTM

**Pull request merged**